core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -58,6 +71,113 @@
         left: 42px;
         position: absolute;
       }
+      @media only screen and (max-width : 600px)
+      {
+        .oppia-collection-player-title-font
+        {
+          font-size: 32px;
+        }
+      }
+      exploration-summary-tile md-card {
```

Comment:
Ah my bad, sorry. OK, please add a comment in the code to explain then (since this is a bit unusual and we should explain the "why" of things), but otherwise this looks fine.